src/watcher.ts

```diff
@@ -45,13 +45,13 @@ function onEntry(entries: SpanielObserverEntry[]) {
     const opts: WatcherCallbackOptions = {
       duration,
       boundingClientRect,
-      visibleTime: entry.time,
+      visibleTime: entry.unixTime,
       intersectionRect
     };
     if (entry.entering) {
       entry.payload.callback(label, opts);
     } else if (entry.label === 'impressed') {
-      opts.visibleTime = entry.time - entry.duration;
+      opts.visibleTime = entry.unixTime - entry.duration;
```

Comment:
We should use `Date.now` for visibleTime that is exposed, to do that we can't use duration. Maybe we should just keep track of visible start time for the entry?

Reply:
good idea, done